components/web/apache2-modules/mod_wsgi/history

```diff
@@ -1,2 +1,4 @@
 web/server/apache-24/module/apache-wsgi-34@4.5.22-2020.0.1.3
 web/server/apache-24/module/apache-wsgi-26@3.4,5.11-2016.0.1.0
+web/server/apache-24/module/apache-wsgi-27@4.9.0,5.11-2021.0.0.0
+web/server/apache-24/module/apache-wsgi-35@4.9.0,5.11-2021.0.0.0
```

Comment:
When you add an FMRI into a history file make sure to increment its COMPONENT_REVISION.
Thus,
web/server/apache-24/module/apache-wsgi-27@4.9.0,5.11-2021.0.0.1

The same applies for wsgi-35.